src/Level.ts

```diff
@@ -0,0 +1,35 @@
+import { t } from 'ttag';
+import SimpleSchema from 'simpl-schema';
+
+import './SimpleSchemaExtensions';
+
+import { LocalizedStringSchema, LocalizedString } from './LocalizedString';
+
+export interface Level {
+  /**
+   * Name of the level
+   */
+  name?: LocalizedString;
+  /**
+   * Index of the level
+   */
+  index?: Number;
+}
+
+export const LevelSchema = new SimpleSchema({
+  name: {
+    type: LocalizedStringSchema,
+    label: t`Level Name`,
+    optional: true,
+    accessibility: {
+      question: t`What the name of the Level`
+    }
+  },
+  index: {
+    type: Number,
+    optional: true,
+    accessibility: {
+      question: t`Index of the level in question`
```

Comment:
```suggestion
      question: t`Numeric index of the level that indicates relative position of this level in relation to other levels (levels with higher indices are assumed to be located above levels with lower indices).

Ground level should have index 0, with levels above ground indicated by positive indices and levels below ground by negative indices.`
```

(Inspired by https://gtfs.org/reference/static/#levelstxt)